cmd/wallet-sdk-gomobile/gomobilewrappers/documentloaderwrapper.go

```diff
@@ -15,14 +16,15 @@ import (
 	"github.com/trustbloc/wallet-sdk/cmd/wallet-sdk-gomobile/api"
 )
 
-// gomobileDocumentLoaderWrapper wraps a gomobile-compatible version of a LDDocumentLoader and translates
+// DocumentLoaderWrapper wraps a gomobile-compatible version of a LDDocumentLoader and translates
 // methods calls to their corresponding Go API versions.
-type gomobileDocumentLoaderWrapper struct {
-	ldDocumentLoader api.LDDocumentLoader
+type DocumentLoaderWrapper struct {
```

Comment:
Exporting this will cause mobile bindings to get generated for this type, which we shouldn't need... Put this in an `internal` folder and `gomobile` will skip it